style.css

```diff
@@ -309,3 +319,217 @@ ul {
   font-size: 24px;
   color: var(--white-color);
 }
+
+.navbar-black {
+  display: none;
+}
+
+.partner {
+  display: none;
+}
+
+footer {
+  display: none;
+}
+
+.invisible {
+  display: none;
+}
+
+/* Desktop version */
+
+@media only screen and (min-width: 768px) {
+  .navbar {
+    background-color: var(--white-color);
+    height: 58px;
+  }
+
+  .navbar-black {
+    display: inline;
+    width: 100%;
+    height: 24px;
+    background-color: var(--dark-color);
+    flex-direction: column;
+  }
+
+  .navbar-black ul {
+    flex-direction: row;
+    display: flex;
+    justify-content: flex-end;
+    gap: 12px;
+    padding: 4px;
+    padding-right: 25px;
+    margin: 0;
+    background-color: var(--dark-color);
+  }
+
+  .navbar-black li {
+    color: var(--white-color);
+  }
+
+  .menu-list {
+    display: flex;
+    gap: 12px;
+  }
+
+  .menu-list :hover {
+    background-color: #d3d3d3;
+    color: var(--primary-color);
+    cursor: pointer;
+  }
+
+  #menu-icon {
+    display: none;
+  }
+
+  .desktop-nav-item {
+    color: var(--dark-color);
+  }
+
+  .fa-bars {
+    display: none;
+  }
+
+  h3 {
+    margin-top: 180px;
+    font-weight: 500;
+    font-size: 48px;
+  }
+
+  h2 {
+    font-weight: 700;
+    font-size: 56px;
+  }
+
+  .hello-section {
+    padding-left: 12%;
+    padding-right: 12%;
+  }
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up: your design is not matching to the microverse provided design so kindly fix this design issue :+1: 
- [ ] Check below :point_down: the comparison between your design and the microverse provided design.

**Your Design**
![Screenshot from 2023-03-23 18-04-21](https://user-images.githubusercontent.com/45226848/227213880-bd7cd3db-5bb6-4a9e-9a61-c2cda3b0dc13.png)

**Provided Design**
![design](https://user-images.githubusercontent.com/50990925/227091473-559ff4b9-928d-48e8-b603-040c56b57c7d.png)